lib/invoicing/ledger_item.rb

```diff
@@ -387,7 +387,8 @@ def acts_as_payment(options={})
 
       scope :exclude_empty_invoices, lambda {
         line_items_assoc_id = ledger_item_class_info.method(:line_items).to_sym
-        line_items_refl = reflections[line_items_assoc_id]
+        line_items_refl = reflections[line_items_assoc_id] ||
```

Comment:
rails 4.2 is not returning `HashWithIndifferentAccess` anymore (for reflections). so, i did a quick fix. this needs cleanup though.